debezium-connector-postgres/src/test/java/io/debezium/connector/postgresql/AbstractRecordsProducerTest.java

```diff
@@ -585,6 +586,14 @@ private long asEpochMicros(String timestamp) {
                              );
     }
 
+    protected List<SchemaAndValueField> schemaAndValuesForIntervalAsString() {
+        //1 year, 2 months, 3 days, 4 hours, 5 minutes, 6 seconds, 78000 ms
+        final String expectedInterval = "P1Y2M3DT4H5M" + 6.78 + "S";
```

Comment:
Why is this concatenation done?

Reply:
@gunnarmorling Well, the `PGInterval` still uses double for seconds expression so I wanted to make sure the double is properly converted due to doubles inexact nature. The experiment has shown that the difference is at 11 position afte decimal point which means it is removevd by the `Interval` scale reducing and the string can be expressed by the real sting value so changing.